examples/rest-api/.env.example

```diff
@@ -0,0 +1,22 @@
+# for OpenAI, Azure OpenAI, etc
+# OPENAI_API_KEY=
+
+# for Azure
+# OPENAI_API_TYPE=
+# OPENAI_API_BASE=
+# OPENAI_API_VERSION=
+
+# for cohere
+# COHERE_API_KEY=
+
+# for anthropic
+# ANTHROPIC_API_KEY=
+
+# for Jina
+# JINACHAT_API_KEY=
+
+# for huggingface
+# HUGGINGFACE_ACCESS_TOKEN=
+
+# for llama2
+# REPLICATE_API_TOKEN=
```

Comment:
This is not going to work since users will do 

```
docker pull <image>
docker run <image> .....
```

So they won't be able to edit the env file.

I am wondering if we should do something like this:

```
docker run -e ENV_VARIABLE1=value1 -e ENV_VARIABLE2=value2 -p 8080:80 your-docker-image:tag
```

Then it will be user's responsibility to pass the relevant docker image whenever they are trying to spawn the docker container.

Reply:
Yup yup, I was just listing down the envs and testing different approaches (`docker-compose`) but for our use case `-e ENV_VAR=value` seems to be the best